packages/nimble-components/src/table-column/anchor/tests/table-column-anchor.spec.ts

```diff
@@ -0,0 +1,473 @@
+        for (const testData of noValueData) {
+            // eslint-disable-next-line @typescript-eslint/no-loop-func
+            it(`displays placeholder string when label ${testData.description}`, async () => {
+                await element.setData(testData.data);
+                await connect();
+                await waitForUpdatesAsync();
+
+                expect(pageObject.getRenderedCellContent(0, 0)).toBe(
+                    'no value'
+                );
+            });
+        }
+
+        it('changing labelFieldName updates display', async () => {
+            await element.setData([{ label: 'foo', otherLabel: 'bar' }]);
+            await connect();
+            await waitForUpdatesAsync();
+
+            const firstColumn = element.columns[0] as TableColumnAnchor;
+            firstColumn.labelFieldName = 'otherLabel';
+            await waitForUpdatesAsync();
+
+            expect(pageObject.getRenderedCellContent(0, 0)).toBe('bar');
+        });
+
+        it('changing placeholder updates display', async () => {
+            await element.setData([{ label: null }]);
+            await connect();
+            await waitForUpdatesAsync();
+
+            const firstColumn = element.columns[0] as TableColumnAnchor;
+            firstColumn.placeholder = 'different value';
+            await waitForUpdatesAsync();
+
+            expect(pageObject.getRenderedCellContent(0, 0)).toBe(
+                'different value'
+            );
+        });
+
+        it('changing data from value to null displays placeholder', async () => {
+            await element.setData([{ label: 'foo' }]);
+            await connect();
+            await waitForUpdatesAsync();
+            expect(pageObject.getRenderedCellContent(0, 0)).toBe('foo');
+
+            await element.setData([{ label: null }]);
+            await waitForUpdatesAsync();
+
+            expect(pageObject.getRenderedCellContent(0, 0)).toBe('no value');
+        });
+
+        it('changing data from null to value displays value', async () => {
+            await element.setData([{ label: null }]);
+            await connect();
+            await waitForUpdatesAsync();
+            expect(pageObject.getRenderedCellContent(0, 0)).toBe('no value');
+
+            await element.setData([{ label: 'foo' }]);
+            await waitForUpdatesAsync();
+
+            expect(pageObject.getRenderedCellContent(0, 0)).toBe('foo');
+        });
+
+        it('when no labelFieldName provided, nothing is displayed', async () => {
+            await connect();
+            await waitForUpdatesAsync();
+
+            const firstColumn = element.columns[0] as TableColumnAnchor;
+            firstColumn.labelFieldName = undefined;
+            await element.setData([{ field: 'foo' }]);
+            await waitForUpdatesAsync();
+
+            expect(pageObject.getRenderedCellContent(0, 0)).toBe('');
+        });
+
+        // Firefox skipped, see: https://github.com/ni/nimble/issues/1075
```

Comment:
Reminder to update this issue to track the new failure (or, even better, make this new test pass on Firefox).

Same for the other instance further down this file.

Reply:
I found the issue and fixed it. I posted a comment on the text column test file explaining why I'm fixing it as part of this change, and what the fix was.